kore/test/Test/Kore/Strategies/AllPath/Verification.hs

```diff
@@ -322,6 +323,46 @@ test_allPathVerification =
         assertEqual ""
             (Left $ Pattern.fromTermLike Mock.e)
             actual
+    , testCase "Priority: should apply first axiom" $ do
```

Comment:
Since this is all-path, would it be more accurate to say `should also apply the first axiom`?